### ISSUE TITLE
[feature request] add a license to this repo

### ISSUE THREAD

**Issue by mueslimak3r** [garrit-schroeder/tv-show-skipper#2]
This is really cool!
I'm interested in forking this and adding a wrapper to interact with the jellyfin API (https://jellyfin.org) so into metadata can be generated for series.

If a license could be added that clarifies whether this would be permitted, that would be great 🙂

This is a good resource for deciding what kind of license you want:
https://choosealicense.com/

**Comment by garrit-schroeder** [comment]
Hi Cameron,
great idea. Do what ever you like. I added the licence too. I hope the one I used suits your case. :) 
Quick points of thought: This is just a prove of concept design. I have no idea if and how well this will work in production. My tests results where promising though. And there is still room for improving the performance too which might be necessary for large libraries in Jellyfin. 

Keep me updated. Or ask me if you have any questions as well.

**Comment by mueslimak3r** [comment]
Thanks!
From my testing it works well. I tried a few similar projects (bw_plex for example) and they were all too complicated to refactor or too slow.

In my fork I've made a branch where I'm converting it to a module that can return a dictionary of paths with their extracted start and end timestamps

Once that's solid I'll make a json exporter and if/when support for intro timestamps is added to the jellyfin API I'll add an interface for that

If you see any changes you're interested in up-streaming I'm happy to clean up my changes and submit PRs
